docs/en/tutorials/extra-lazy-associations.rst

```diff
@@ -19,6 +19,7 @@ can be called without triggering a full load of the collection:
 -  ``Collection#count()``
 -  ``Collection#get($key)``
 -  ``Collection#slice($offset, $length = null)``
+-  ``Collection#isEmpty()``
```

Comment:
The current list seems to be using alphabetical order. Can you please move this to the right place? Also, according to https://github.com/doctrine/orm/issues/11208 you should target 2.20

Reply:
done at https://github.com/doctrine/orm/pull/11716